async_upnp_client/const.py

```diff
@@ -63,6 +63,7 @@
     "float": {"type": float, "in": float, "out": str},
     "char": {"type": str, "in": str, "out": str},
     "string": {"type": str, "in": str, "out": str},
+    "xml": {"type": ET.Element, "in": ET.Element, "out": lambda tree: tree},
```

Comment:
Shouldn't the `out` be `lambda tree: ET.dump(tree)`? Although that might cause problems later on regarding encoding and/or escaping and such.

Reply:
I see the special handling later on. Perhaps add a comment on this line regarding the special handling.

Reply:
In my tests just the lambda solve the problem.